iocore/eventsystem/SocketManager.cc

```diff
@@ -108,3 +111,36 @@ SocketManager::close(int s)
   } while (res == -EINTR);
   return res;
 }
+
+bool
+SocketManager::fastopen_supported()
```

Comment:
Or maybe embedded in the regression test as a file scoped function rather than a method?